troubleshooting.md

```diff
@@ -0,0 +1,551 @@
+
+* [Following the Installation](#following-the-installation) 
+
+* [Node Related Issues](#node-related-issues)
+
+* [NTP out of sync](#ntp-out-of-sync)
+
+* [Registry Issues](#registry-issues) 
+
+* [IPv6 Disconnected Install Troubleshooting](#ipv6-disconnected-install-troubleshooting)
+
+* [Misc Issues](#misc-issues) 
+
+
+# Bootstrap VM related issues 
+
+* Cannot login to the Bootstrap VM
+```
+$ ssh core@172.22.0.2
+Permission denied (publickey,password,keyboard-interactive).
+```
+
+When attempting to access the Bootstrap VM and `permissioned_denied` error occurs, verify that your SSH key for the user attempting to log into the VM is set within your `install-config.yaml` file. 
+
+
+After the installation is triggered, Wait for few minutes and check/verify bootstrap VM is up and running from Provisioner node: 
+
+```
+[kni@provisioner]$ sudo virsh list
+ Id    Name                           State
+----------------------------------------------------
+ 12    kni7-xf6fq-bootstrap           running
+```
+
+If there is no sign of Bootstrap VM running on the provioner node, check if the libvirtd is running(if not, start the service) on the provisioner node. 
+
+```
+systemctl status libvirtd
+● libvirtd.service - Virtualization daemon
+   Loaded: loaded (/usr/lib/systemd/system/libvirtd.service; enabled; vendor preset: enabled)
+   Active: active (running) since Mon 2020-02-03 13:08:51 EST; 2 weeks 0 days ago
+     Docs: man:libvirtd(8)
+```
+
+If re-running the deployment, please make sure to delete all the older VM including its volume as shown below, if not the bootstrap VM will not get a IP of `172.22.0.2` through provisioning dnsmasq and causes failure in further deployment process:
+
+
+Clean up the VM and its volume with the help of below script. 
+
+```
+BOOTSTRAP=$(virsh list --all | grep "$CLUSTER.*bootstrap" | tail -1 | awk '{print $2}')
+virsh destroy $BOOTSTRAP
+virsh undefine $BOOTSTRAP
+virsh vol-delete $BOOTSTRAP default
+virsh vol-delete $BOOTSTRAP.ign default
+```
+
+* You will get the IP of this VM by issuing `sudo virsh net-dhcp-leases` usually always the private IP of this VM is `172.22.0.2` , you can login to this VM using the private SSH key with core as its user:
+
+```
+[kni@provisioner ~]$ sudo virsh list 
+
+ Id    Name                           State
+----------------------------------------------------
+ 23    kni7-4q49b-bootstrap           running
+[kni@provisioner ~]$ sudo virsh console kni7-4q49b-bootstrap
+Connected to domain kni7-4q49b-bootstrap
+Escape character is ^]
+
+Red Hat Enterprise Linux CoreOS 43.81.202001142154.0 (Ootpa) 4.3
+SSH host key: SHA256:BRWJktXZgQQRY5zjuAV0IKZ4WM7i4TiUyMVanqu9Pqg (ED25519)
+SSH host key: SHA256:7+iKGA7VtG5szmk2jB5gl/5EZ+SNcJ3a2g23o0lnIio (ECDSA)
+SSH host key: SHA256:DH5VWhvhvagOTaLsYiVNse9ca+ZSW/30OOMed8rIGOc (RSA)
+ens3:  fd35:919d:4042:2:c7ed:9a9f:a9ec:7
+ens4: 172.22.0.2 fe80::1d05:e52e:be5d:263f
+localhost login: 
+```
+
+# ssh -i ~/.ssh/id_rsa core@172.22.0.2
```

Comment:
this is a header but shouldn't be

Reply:
removed the header and placed it under a terminal code snippet format.